plugin/godoc.vim

```diff
@@ -65,6 +65,14 @@ function! s:GodocView()
   au BufHidden <buffer> call let <SID>buf_nr = -1
 endfunction
 
+function! s:GodocNotFound(content)
+  if !len(a:content)
+    return 1
+  endif
+
+  return a:content =~# '^.*: no such file or directory\n'
```

Comment:
You might want to add this fix too: https://github.com/fatih/vim-go/pull/436 fyi